hw2/04-background.html

```diff
@@ -6,24 +6,25 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
   <body>
-    <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
-      </div>
+    <main class="container mt-5">
+      <section class="bg-light border rounded p-3">
+        <h1 class="card-title mt-2">Exercise 04 - Background</h1>
+        <p class="mt-4">
+          The background color will change every few seconds (as indicated
+          below). You can stop this behavior and you can also change how often
+          the background color changes.
+        </p>
+        <div>
+          <label>Background change interval (seconds): </label>
+          <input type="number" id="seconds" min="1"/>
+        </div>
+        <input class="fw-lighter mt-3 text-light bg-primary border rounded" type="button" value="Start" id="startstop"></input>
+      </section>
```

Comment:
WAVE flagged button for contrast errors. This can be fixed by just using class="btn btn-primary" and for stop class="btn btn-danger"